composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Lcobucci\\Clock\\": "src"
```

Comment:
This is the same as `autoload`, so probably either not needed, or a mistake.

Reply:
Ouch, that was indeed a mistake.